code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -843,23 +843,20 @@
 	return ..()
 
 /datum/species/diona/handle_life(var/mob/living/carbon/human/H)
+	H.radiation = Clamp(H.radiation, 0, 100) //We have to clamp this first, then decrease it, or there's a few edge cases of massive heals if we clamp and decrease at the same time.
```

Comment:
The reason this isn't below `var/rads = H.radiation / 25` and `H.radiation = Clamp(H.radiation-rads, 0, 100)` is because there's a few weird edge cases where you can technically get something like 160 radiation, resulting in a heal that is beyond the maximum allowed of 4, so we have to clamp it to the max/min values here FIRST, then decrease the rads.